src/søknad/steg/5-arbeidssituasjon/underUtdanning/TidligereUtdanning.tsx

```diff
@@ -0,0 +1,130 @@
+import LocaleTekst from '../../../../language/LocaleTekst';
+import FeltGruppe from '../../../../components/gruppe/FeltGruppe';
+import JaNeiSpørsmål from '../../../../components/spørsmål/JaNeiSpørsmål';
+import KnappBase from 'nav-frontend-knapper';
+import SeksjonGruppe from '../../../../components/gruppe/SeksjonGruppe';
+import Utdanning from './Utdanning';
+import { Element, Undertittel } from 'nav-frontend-typografi';
+import { hentTekst } from '../../../../utils/søknad';
+import { ISpørsmål } from '../../../../models/spørsmal';
+import { linjeKursGrad, utdanningEtterGrunnskolenSpm } from './UtdanningConfig';
+import { useIntl } from 'react-intl';
+import { tomPeriode } from '../../../../utils/søknadsfelter';
+
+interface Props {
+  underUtdanning: IUnderUtdanning;
+  settUnderUtdanning: (utdanning: IUnderUtdanning) => void;
+}
+const TidligereUtdanning: React.FC<Props> = ({
+  underUtdanning,
+  settUnderUtdanning,
+}) => {
+  const intl = useIntl();
+  const linjeKursGradLabel = hentTekst(linjeKursGrad.label_tekstid, intl);
+  const tomUtdanning: IUtdanning = {
+    linjeKursGrad: { label: linjeKursGradLabel, verdi: '' },
+    periode: tomPeriode,
+  };
+
+  const [tidligereUtdanning, settTidligereUtdanning] = useState<IUtdanning[]>([
+    tomUtdanning,
+  ]);
+
+  useEffect(() => {
+    underUtdanning.harTattUtdanningEtterGrunnskolen &&
+      settUnderUtdanning({
+        ...underUtdanning,
+        tidligereUtdanning: tidligereUtdanning,
+      });
+
+    // eslint-disable-next-line
+  }, [tidligereUtdanning]);
+
+  const leggTilUtdanning = () => {
+    const nyUtdanning: IUtdanning = tomUtdanning;
+    const allUtdanning: IUtdanning[] = tidligereUtdanning;
+    allUtdanning.push(nyUtdanning);
+    settUnderUtdanning({ ...underUtdanning, tidligereUtdanning: allUtdanning });
+  };
+
+  const settHarTattUtdanningEtterGrunnskolen = (
+    spørsmål: ISpørsmål,
+    svar: boolean
+  ) => {
+    const tattUtdanningEtterGrunnskolenFelt = {
+      label: hentTekst(spørsmål.tekstid, intl),
+      verdi: svar,
+    };
+
+    if (!svar && underUtdanning.tidligereUtdanning) {
+      const endretUtdanning = underUtdanning;
+      delete endretUtdanning.tidligereUtdanning;
+      settUnderUtdanning({
+        ...endretUtdanning,
+        [spørsmål.søknadid]: tattUtdanningEtterGrunnskolenFelt,
+      });
+    } else if (svar && !underUtdanning.tidligereUtdanning) {
+      settUnderUtdanning({
+        ...underUtdanning,
+        tidligereUtdanning: [tomUtdanning],
+      });
+    } else {
+      underUtdanning &&
+        settUnderUtdanning({
+          ...underUtdanning,
+          [spørsmål.søknadid]: tattUtdanningEtterGrunnskolenFelt,
+        });
+    }
+  };
+
+  return (
+    <SeksjonGruppe>
+      <KomponentGruppe>
+        <Undertittel className={'sentrert'}>
+          <LocaleTekst tekst={'utdanning.tittel.tidligere'} />
+        </Undertittel>
+      </KomponentGruppe>
+
+      <KomponentGruppe>
+        <JaNeiSpørsmål
```

Comment:
Hmm, her må jeg trykke to ganger på "Ja" før det skjer noe 🤔 

![Skjermbilde 2020-03-05 kl  14 01 38](https://user-images.githubusercontent.com/1413265/75984154-29828800-5eea-11ea-863a-12f75b558e50.png)

Reply:
Noo, det var en bug jeg trodde jeg hadde klart å fikse 😫

Reply:
Fixed i ny commit!

Reply:
Smooth!